src/http/params.cr

```diff
@@ -221,7 +221,7 @@ module HTTP
     # *default* value when there is no such param.
     #
     # ```
-    # params.fetch("email", "none@example.org")           # => "john@example.org"
+    # params.fetch("email", "none@example.org")           # => "none@example.org"
```

Comment:
This is losing nuance of the example. It seems like the idea is that `params` actually contains `"john@example.org"` as the `"email"`, but you instead are hitting the fallback to the default (in both lines now).

Reply:
Oh, I missed the intent of the code. You are right.

Unfortunately, line 193 overrides the `"params"` and the `"email"` key no longer exists. 
https://github.com/crystal-lang/crystal/blob/master/src/http/params.cr#L193

I'm going to set `"email"` at the beginning of this block, and then change `"none"` back to `"john"`.